Docs: updated quick-start (#2589)
liqotech/liqo: docs/examples/quick-start.md

```text
NAME                                       READY   STATUS    RESTARTS   AGE
liqo-controller-manager-6888ccc645-hnxrl   1/1     Running   0          8m15s
liqo-crd-replicator-5f5b448bd-ldz67        1/1     Running   0          8m15s
liqo-fabric-28tb5                          1/1     Running   0          8m15s
liqo-fabric-dvjgk                          1/1     Running   0          8m15s
liqo-ipam-658dcbcb66-z4vhq                 1/1     Running   0          8m15s
liqo-metric-agent-597c5dbcfd-bzg2t         1/1     Running   0          8m15s
liqo-proxy-599958d9b8-6fzfc                1/1     Running   0          8m15s
liqo-webhook-8fbd8c664-pxrfh               1/1     Running   0          8m15s
```

> 

Reviewer: ```suggestion
```

## Peer two clusters

Once Liqo is installed in your clusters, you can establish new *peerings*.
In this example, we will leverage the *liqoctl peer* command to peer the two clusters.

> This approach requires the user to have access to the kubeconfig of both clusters.

Reviewer: ```suggestion
This approach requires the user to have access to the `kubeconfig` of both clusters.
```

Let's issue the peering command from the consumer cluster, which is Rome in this case.

> The `--remote-kubeconfig` flag is used to specify the kubeconfig of the remote provider cluster, which is Milan in this case.

Reviewer: ```suggestion
The `--remote-kubeconfig` flag is used to specify the `kubeconfig` of the remote provider cluster, which is Milan in this case.
```

```bash
liqoctl peer --remote-kubeconfig "$KUBECONFIG_MILAN" --server-service-type NodePort
```

> The peering should be completed successfully after a few seconds or minutes, depending on the downloading time of the new images.

Reviewer: ```suggestion
The peering should be completed successfully after a few seconds (if the Docker images are already cached in the involved clusters) or minutes (when the Docker images have to be downloaded).
```

```

At steady-state, you should see an output similar to the following.

> Different pods may be hosted by either the local nodes (*rome-worker* in the example below) or remote cluster (*milan* in the example below), depending on the scheduling decisions.

Reviewer: ```suggestion
Pods may be hosted by either the local nodes (*rome-worker* in the example below) or the remote cluster (*milan* in the example below), depending on the decisions of the Kubernetes scheduler.
```
